CONTRIBUTING.md

```diff
@@ -24,6 +24,12 @@ standardized:
   - For blog posts, place the file in `_posts/<language code>/`
 - Set the `lang` field to `<language code>`
 - Append `-<language code>` to both the `slug` and `name` fields
+- When linking internally, link to the appropriate translation, if one is available.
```

Comment:
s/linking internally/linking to material on the Optech site/ for clarity.

Reply:
Changed.